extensions/vscode/assets/icons/template/mapping.json

```diff
@@ -0,0 +1,3 @@
+{
+  "publish": 61697
```

Comment:
This was generated without the template, but figured including the mapping would make it easier to reference if used in the `package.json` for example.